ft-sdk/src/layout.rs

```diff
@@ -19,6 +19,22 @@ where
         Self: Sized;
 }
 
+pub trait ActionWithLog<L, E>
```

Comment:
Nopes. Action without log is non sense. It's like creating person with head. You treat action without log same way you deal with person without head, bury it in the ground, don't preserve them in your home (repo).

Reply:
It will be replaced by Action later. I created this since I needed one for logging on one specific action without breaking other action routes.

Reply:
Even then this approach won't work, or is not ideal.